timeseries/src/autogluon/timeseries/models/abstract/abstract_timeseries_model.py

```diff
@@ -764,6 +939,32 @@ def convert_to_refit_full_via_copy(self) -> Self:
 
         return refit_model
 
+    def convert_to_refit_full_template(self):
+        """
+        After calling this function, returned model should be able to be fit without X_val, y_val using the iterations trained by the original model.
+
+        Increase max_memory_usage_ratio by 25% to reduce the chance that the refit model will trigger NotEnoughMemoryError and skip training.
+        This can happen without the 25% increase since the refit model generally will use more training data and thus require more memory.
+        """
+        params = copy.deepcopy(self.get_params())
+
+        if "hyperparameters" not in params:
+            params["hyperparameters"] = dict()
+
+        if AG_ARGS_FIT not in params["hyperparameters"]:
+            params["hyperparameters"][AG_ARGS_FIT] = dict()
+
+        # Increase memory limit by 25% to avoid memory restrictions during fit
+        params["hyperparameters"][AG_ARGS_FIT]["max_memory_usage_ratio"] = (
+            params["hyperparameters"][AG_ARGS_FIT].get("max_memory_usage_ratio", 1.0) * 1.25
+        )
```

Comment:
TS models do not use `max_memory_usage_ratio` [here](https://github.com/autogluon/autogluon/pull/4909/files#diff-72dcfbecb24aa875b49f6e776b10cbe58ce394cb5f13f884d7473caa89e6e168L183). Do we still need to keep that here?

Reply:
added a todo 👍